src/Queue/QueueBase.php

```diff
@@ -117,7 +117,7 @@ protected function getQueue(AMQPChannel $channel, array $options = []) {
 
     // Allow modules to override queue settings.
     if (isset($queue_info[$this->name])) {
-      $queue_options += $queue_info[$this->name];
+      $queue_options = $queue_info[$this->name];
     }
```

Comment:
You're right there is an error, but the fix needs to use array_merge instead : using += on an array won't overwrite existing keys, preventing the right member for overriding the default if it is already set.